libraries/lib-mixer/WideSampleSequence.h

```diff
@@ -80,13 +82,14 @@ class MIXER_API WideSampleSequence
     @copydetails SampleTrack::GetFloats()
     @param format sample format of the destination buffer
     */
-   virtual bool Get(size_t iChannel, size_t nBuffers, samplePtr buffers[],
-      sampleFormat format, sampleCount start, size_t len,
+   virtual bool Get(
+      size_t iChannel, size_t nBuffers, samplePtr buffers[],
+      sampleFormat format, sampleCount start, size_t len, bool backward,
```

Comment:
Good, now please add to the Doxygen comments above, explaining the argument, and the expectation that the outputs are reversed.

And likewise for `GetEnvelopeValues`